media/base/media_log.h

```diff
@@ -35,6 +35,39 @@
 
 namespace media {
 
+#if defined(STARBOARD)
+
+// A stub MediaLog implementation.
+// TODO(b/230916218): Consider making use of MediaLog for debugging.
+class MEDIA_EXPORT MediaLog {
```

Comment:
Are we able to use MediaLog as is now?  It was commented out due to //base was on an old version.  Feel free to create a bug to revisit after merging this PR.

Reply:
MediaLog is now enabled as-is

Reply:
While enabling MediaLog works for Cobalt, it breaks the media_sandbox target. I left the stub implementation in for now.